src/platforms/node/guides/express/performance/index.mdx

```diff
@@ -3,7 +3,7 @@ title: Performance Monitoring
 description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
 ---
 
-By default, Sentry error events will not get trace context unless you configure the scope with the transaction, as illustrated in the code sample below.
+When integrating Sentry with an Express application, you can leverage provided middlewares to automatically instrument the performance of your application.
```

Comment:
l: To me "instrument the performance of.." sounds a little off. What about

```suggestion
When integrating Sentry with an Express application, you can leverage provided middlewares to automatically instrument and monitor the performance of your application.
```

?